frontend/package.json

```diff
@@ -54,18 +54,18 @@
   "dependencies": {
     "@openshift/dynamic-plugin-sdk": "^4.0.0",
     "@openshift/dynamic-plugin-sdk-utils": "^4.0.1",
-    "@patternfly/patternfly": "^5.2.1",
+    "@patternfly/patternfly": "^5.3.1",
     "@patternfly/quickstarts": "^5.1.0",
```

Comment:
Upgrade quickstart to 5.3.0 https://www.npmjs.com/package/@patternfly/quickstarts